CONTRIBUTING.md

```diff
@@ -41,27 +41,27 @@ to know that you have a clean state.
   ```sh
   # if your patch is browser specific
   # (e.g. upload files)
-  grunt test:desktop
+  npm run test:desktop
 
   # if your patch is mobile specific
   # (e.g. flick or swipe tests)
-  grunt test:ios test:android
+  npm run test:mobile
```

Comment:
oh now I see that you added that command. I am not sure about this. Most people only have ios or android set up. Only the hardcore appium tester have both I would assume. So I am fine with having the npm command but let's make sure we also show to run ios or android as single test